templates/commands/goldentest/verify.go

```diff
@@ -124,7 +140,7 @@ func (c *VerifyCommand) Run(ctx context.Context, args []string) (rErr error) {
 		if err != nil {
 			if common.IsStatNotExistErr(err) {
 				return fmt.Errorf("no recorded test data in %q, "+
-					"please run `record` command to record the template rendering result to golden tests", goldenDataDir)
+					"please run `recordTestCases` command to recordTestCases the template rendering result to golden tests", goldenDataDir)
```

Comment:
I think this might be a find-and-replace error. I don't think the `recordTestCases` command exists. So `recordTestCases` should be changed back to `record`.